_data/team/roles/core_library_maintenance.yml

```
@@ -0,0 +1,50 @@
+role: Core library maintenance
+description: |
+  The core library maintenance team is responsible for the
+  managing the maintenance of the MDAnalysis library.
+
+subroles:
+  - subrole: Issue management
+    tasks:
+      - Initial triage and tagging of issues
+      - Managing timely responses and resolving issues
+    current_leads:
+      - Rocco Meli
+    current_members:
+      - All core devs
+    historical_members:
+      - Oliver Beckstein
+      - Irfan Alibay
+      - Jonathan Barnoud
+    
+
+  - subrole: Pull request management
+    tasks:
+      - Reviewing, shepherding, and merging pull requests
+    current_leads:
+      - Rocco Meli
+      - Oliver Beckstein
+    current_members:
+      - All core devs
+    historical_members:
+      - Oliver Beckstein
+      - Richard Gowers
+      - Irfan Alibay
+      - Hugo MacDermott-Opeskin
+      - Jonathan Barnoud
+    
+  
+  - subrole: General maintenance
+    tasks:
+      - Standards compliange (e.g. managing metadata such as the author list)
+      - Tracking new dependencies
+      - Emergency fixes
+      - Other emergency maintenance
```

```suggestion
      - Other general maintenance tasks
```
I think we can agree that _a lot_ of the maintenance tasks weren't "emergencies" they just needed to be done.